compiler/parser/parser.cpp

```diff
@@ -414,7 +414,19 @@ static void ensureRequiredStandardModulesAreParsed() {
       UnresolvedSymExpr* oldModNameExpr = toUnresolvedSymExpr(moduleExpr);
 
       if (oldModNameExpr == NULL) {
-        continue;
+        // Allow us to find symbols due to nested references
```

Comment:
The term "nested reference" doesn't really mean much to me intuitively.  I think the intent of this comment would be clearer if it said something like "Handle the case of `[use|import] Mod.symbol` by extracting `Mod` from the `Mod.symbol` expression."